.github/workflows/pythonpackage.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
-        platform: [ubuntu-latest, macos-latest, windows-latest]
+        platform: [ubuntu-20.04, macos-latest, windows-latest]
```

Comment:
Is there any reason we need to pin it to a certain version?

Reply:
No worries and thanks for reviewing! The CI workflow `actions/setup-python@v4` kept failing on `ubuntu-latest` with:

`Error: Version 3.6 with arch x64 not found`

This error was reported in

https://github.com/actions/setup-python/issues/544 

and the recommended fix is to hard code to Ubuntu 20.04 if Python 3.6 is needed since they no longer are supporting Python 3.6.

I figured this could be acceptable until we sunset Python 3.6.

Reply:
ah, got it. Thanks for replying! I'm now encountering the same issue on my own PR haha

Reply:
Ya, it just started happening. The team didn’t make tarballs of 3.6 for the new distro and chose not to add them since 3.6 is already deprecated.

Side note, it’s usually best to specify specific versions rather than latest for this reason. You get more repeatable results that way, so it might be good to do the same for the other OS’s.